back/sssscs/src/main/java/com/ib/pki/manual/KeyUtil.java

```diff
@@ -0,0 +1,155 @@
+package com.ib.pki.manual;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.SecureRandom;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Base64;
+import java.util.Date;
+
+import org.bouncycastle.asn1.x500.X500Name;
+import org.bouncycastle.asn1.x500.X500NameBuilder;
+import org.bouncycastle.asn1.x500.style.BCStyle;
+import org.bouncycastle.openssl.jcajce.JcaMiscPEMGenerator;
+import org.bouncycastle.openssl.jcajce.JcaPEMWriter;
+import org.bouncycastle.util.io.pem.PemObjectGenerator;
+
+public class KeyUtil {
+	private static String DIR_KEYS = "./keys";
+	private static String DIR_CERTS = "./certs";
+	private static String SECRET = "HFJKEHF2-r-2dhUOI#_R0238r0-fwk3w=e2u8v f3h";
+	
+	////////////////////////////////////////////////////////////
+	// Filename generators
+	////////////////////////////////////////////////////////////
+
+	private String getFnameCert(String alias) {
+		return DIR_CERTS + "/" + alias + ".crt";
+	}
+	
+	private String getFnamePrivKey(String alias) {
+		return DIR_KEYS + "/" + alias + ".key";
+	}
+	
+	////////////////////////////////////////////////////////////
+	// Read and write keys/certs.
+	////////////////////////////////////////////////////////////
+	
+	public PrivateKey getPrivateKey(String alias) {
+		String fnamePriv = getFnamePrivKey(alias);
+		try {
+			if (!new File(fnamePriv).isFile()) {
+				return null;
+			}
+			FileInputStream is = new FileInputStream(fnamePriv);
+			byte[] key = Base64.getDecoder().decode(is.readAllBytes());
+	        KeyFactory keyFactory = KeyFactory.getInstance("RSA");
+	        PKCS8EncodedKeySpec keySpec = new PKCS8EncodedKeySpec(key);
+	        PrivateKey finalKey = keyFactory.generatePrivate(keySpec);
+			
+	        return finalKey;
```

Comment:
Bad indentation in try block

Reply:
Fixed in [b111018](https://github.com/magley/sssscs/pull/5/commits/b111018d31a0b6a52e6ee80782ef789c34c47a82).